computer_vision/fast_corner.py

```diff
@@ -0,0 +1,34 @@
+import cv2 as cv
+
+"""
+Features from accelerated segment test (FAST) Corner Detector
+https://en.wikipedia.org/wiki/Features_from_accelerated_segment_test
+"""
+
+
+def fast_detector(img):
+    fast = cv.FastFeatureDetector_create()
+    """
+    Drawing keypoints
+    """
```

Comment:
These are comments, and should be indicated with the python comment token `#`
```suggestion
    # Drawing keypoints
```

Reply:
Noted, will update :)